webextensions/api/menus.json

```diff
@@ -148,7 +148,7 @@
             "__compat": {
               "support": {
                 "chrome": {
-                  "version_added": "≤58",
+                  "version_added": "44",
```

Comment:
:x: (Like `page_action`) In my opinion this landed in https://github.com/chromium/chromium/commit/69e1c12d5b212fc29af5f31982ccca8cb66e5c01, i.e. [38.0.2123.0](https://github.com/chromium/chromium/releases/tag/38.0.2123.0), before Chrome [38(0.2125)](https://duckduckgo.com/?t=ffab&q=chrome+38.0&ia=web) was released, so it should be in Chrome 38.